templates/security/security.phtml

```diff
@@ -57,8 +57,8 @@
                     A security advisory will be released on the Laminas Project
                     site detailing the vulnerability, as well as
                     recommendations for end-users to protect themselves.
-                    Security advisories will be listed at <a href="/security/advisories">https://getlaminas.org/security/advisories</a>,
-                    as well as via a <a href="/security/feed">feed</a> (which
+                    Security advisories will be listed at <a href="<?= $this->url('security.pages', ['action' => 'advisories']) ?>">https://getlaminas.org/security/advisories</a>,
```

Comment:
To output the full URL the option `force_canonical` can be used:

https://docs.laminas.dev/laminas-view/helpers/url/#fully-qualified-domain-name

Reply:
OMG, I had no idea that feature existed! I've been doing all sorts of custom helpers to accomplish this!

Reply:
Oh, I knew that feature exists. But as there is a `mezzio` "url" helper aswell (which does not support that option) I did not add it. Thats kinda confusing tho.

Reply:
And thats actually which is being used, as we are not using laminas-view but the platesrenderer.

https://github.com/mezzio/mezzio-helpers/blob/5.3.0/src/UrlHelper.php#L112

The UrlHelper only provides the path.

But I can combine with the `ServerUrlHelper` instead to call 2 functions tho.